src/components/ChatInput/ChatInput.js

```diff
@@ -163,6 +164,17 @@ const ChatInput = () => {
     }
   };
 
+  const parseEmoji = (text) => {
```

Comment:
great work, please take this function inside - `lib/emoji.js` (create one)

Reply:
Done sir 👍